mlrun/datastore/datastore_profile.py

```diff
@@ -367,15 +367,15 @@ def safe_literal_eval(value):
             )
 
 
-def datastore_profile_read(url):
+def datastore_profile_read(url, project_name=""):
     parsed_url = urlparse(url)
     if parsed_url.scheme.lower() != "ds":
         raise mlrun.errors.MLRunInvalidArgumentError(
             f"resource URL '{url}' cannot be read as a datastore profile because its scheme is not 'ds'"
         )
 
     profile_name = parsed_url.hostname
-    project_name = parsed_url.username or mlrun.mlconf.default_project
+    project_name = project_name or mlrun.mlconf.default_project
```

Comment:
Are you sure we don't want to keep `parsed_url.username` as an option?

Reply:
I am.  This was supposed to be a way to pass project name , and now we remove it (see https://jira.iguazeng.com/browse/ML-5634)